sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -357,7 +357,7 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
    * }}}
    */
   override def visitManageResource(ctx: ManageResourceContext): LogicalPlan = withOrigin(ctx) {
-    val mayebePaths = remainder(ctx.identifier).trim
+    val mayebePaths = pathWrapper(remainder(ctx.identifier).trim)
```

Comment:
Can we use string literal or identifier rule in ANTLER rather than manually fixing here from identifer? BTW, does Hive support that case as well?

Reply:
Yea, that approach @HyukjinKwon addressed above looks nicer to me.

Reply:
I changed the grammar to take string literal as well for ADD/LIST. 

Now all of  `add file abc.txt`, `add file 'abc.txt'` and `add file "abc.txt"`  are supported.